.github/buildomat/jobs/a4x2-deploy.sh

```diff
@@ -3,7 +3,6 @@
 #: name = "a4x2-deploy"
 #: variety = "basic"
 #: target = "lab-2.0-opte-0.27"
-#: rust_toolchain = "stable"
```

Comment:
Looks like the other a4x2 job uses a specific toolchain version in lieu of stable. Did you intend to delete this and pin to 1.77.2, like `a4x2-prepare.sh`?

Reply:
a4x2-deploy is similar to the deploy job in that neither needs a Rust toolchain, because they're both targeting lab environments where the ramdisk image does not contain any compilers. (Both these jobs get all their artifacts compiled on a job with a non-lab target.)